core/common/src/ContextRealityModel.ts

```diff
@@ -234,6 +241,10 @@ export class ContextRealityModel {
   public readonly description: string;
   /** An optional identifier that, if present, can be used to elide a request to the reality data service. */
   public readonly realityDataId?: string;
+  /** If true, reality model is not drawn.
+   * @beta
+  */
+  public readonly invisible?: boolean;
```

Comment:
You probably want to be able to change this, right (not `readonly`)? If so we will want the setter to raise an event so we can, e.g., invalidate a viewport's scene when the visibility changes.

Reply:
I had the same thought process as you but then realized this can be achieved by calling `ContextRealityModels.update`, or maybe I'm missing something and that's a terrible idea?

Reply:
Not terrible but more awkward. ContextRealityModel already has 3 mutable properties with corresponding events anyway.

Reply:
alright, implemented a new event.